diff --git b/lisp-src/bundle/software/clack-20210807-git/src/handler/toot.lisp a/lisp-src/bundle/software/clack-20210807-git/src/handler/toot.lisp
index 33df645..0a03868 100644
--- b/lisp-src/bundle/software/fukamachi-clack-20211109013733/src/handler/toot.lisp
+++ a/lisp-src/bundle/software/fukamachi-clack-20211109013733/src/handler/toot.lisp
@@ -80,7 +80,8 @@ before pass to Clack application."
       (list
        :request-method (request-method req)
        :script-name ""
-       :path-info (let ((flex:*substitution-char* #-abcl #\Replacement_Character
+       :path-info (let ((flex:*substitution-char* #-(or abcl lispworks) #\Replacement_Character
+                                                  #+lispworks #\?
                                                   #+abcl #\?))
                     (url-decode (request-path req)))
        :server-name server-name
