diff --git b/lisp-src/bundle/software/clack-20210807-git/src/handler/toot.lisp a/lisp-src/bundle/software/clack-20210807-git/src/handler/toot.lisp
index 33df645..0a03868 100644
--- b/lisp-src/bundle/software/fukamachi-clack-20211109013733/src/handler/toot.lisp
+++ a/lisp-src/bundle/software/fukamachi-clack-20211109013733/src/handler/toot.lisp
@@ -80,7 +80,8 @@ before pass to Clack application."
       (list
        :request-method (request-method req)
        :script-name ""
-       :path-info (let ((flex:*substitution-char* #-abcl #\Replacement_Character
+       :path-info (let ((flex:*substitution-char* #-(or abcl lispworks) #\Replacement_Character
+                                                  #+lispworks #\?
                                                   #+abcl #\?))
                     (url-decode (request-path req)))
        :server-name server-name
diff --git b/lisp-src/bundle/software/toot-20121125-git/http.lisp a/lisp-src/bundle/software/toot-20121125-git/http.lisp
index 2d95201..b5ca37c 100644
--- b/lisp-src/bundle/software/gigamonkey-toot-20190319051048/http.lisp
+++ a/lisp-src/bundle/software/gigamonkey-toot-20190319051048/http.lisp
@@ -333,12 +333,14 @@ different thread than accept-connection is running in."
 
 (defun setup-ssl-stream (acceptor stream)
   ;; attach SSL to the stream if necessary
-  (with-slots (ssl-certificate-file ssl-private-key-file ssl-private-key-password) acceptor
-    (cl+ssl:make-ssl-server-stream
-     stream
-     :certificate ssl-certificate-file
-     :key ssl-private-key-file
-     :password ssl-private-key-password)))
+  (error "SSL support was removed from Toot")
+;;  (with-slots (ssl-certificate-file ssl-private-key-file ssl-private-key-password) acceptor
+;;    (cl+ssl:make-ssl-server-stream
+;;     stream
+;;     :certificate ssl-certificate-file
+;;     :key ssl-private-key-file
+;;     :password ssl-private-key-password))
+)
 
 (defun finish-response-body (request)
   (with-slots (content-stream) request
diff --git b/lisp-src/bundle/software/toot-20121125-git/toot.asd a/lisp-src/bundle/software/toot-20121125-git/toot.asd
index 05987ab..43b42d0 100644
--- b/lisp-src/bundle/software/gigamonkey-toot-20190319051048/toot.asd
+++ a/lisp-src/bundle/software/gigamonkey-toot-20190319051048/toot.asd
@@ -46,7 +46,7 @@ output.")
                :cl-fad
                :cl-ppcre
                :flexi-streams
-               :cl+ssl
+               ;; :cl+ssl
                :md5
                :trivial-backtrace
                :usocket
